README.md

```diff
@@ -13,39 +13,37 @@ zigpy contains common code implementing Zigbee ZCL, ZDO and application state ma
 
 Reference implementation of the zigpy library exist in **[Home Assistant](https://www.home-assistant.io)** (Python based open source home automation software) as part of its **[ZHA integration component](https://www.home-assistant.io/integrations/zha/)**.
 
-### Zigbee device OTA updates
-
-zigpy have code to automatically download and perform OTA (Over-The-Air) firmware updates of Zigbee devices if the OTA firmware image provider source URL for updates is known. The OTA update directory is optional and it can also be used for any offline firmware files that you provide yourself.
-
-Online OTA providers for firmware updates are currently only available for IKEA and LEDVANCE devices. OTA updates for device of other manufactures could possible also be supported by zigpy in the future, if these manufacturers publish their device OTA firmware images publicly.
+## Compatible Zigbee coordinator hardware
 
-## How to install and test, report bugs, or contribute to this project
+Radio libraries for zigpy are separate projects with their own repositories and implanting the support for the hardware radios modules.
+Stabile compatible radio labaries: 
```

Comment:
```suggestion
Stable radio labaries: 
```

Reply:
`labaries` -> `libraries` ?

Reply:
this whole thing needs to be ran through grammarly.